### DISCUSSION TITLE
chezmoi no encyption error

### DISCUSSION THREAD

**Discussion by Archie-2021** [twpayne/chezmoi#3558] [Q&A]
I am setting up my config on an new computer which is not trusted so I dont want  decryption of my gpg key. But no matter what i put in my chezmoi config file it gives me this error. I have done similar setup before . I am sure I am making a basic mistake but cant find it after tshooting for hour s

**Comment by halostatue** [comment]
In order to help you diagnose this better, we need the same information that would be found making a [support issue](https://github.com/twpayne/chezmoi/issues/new?assignees=&labels=support&projects=&template=01_support_request.md&title=).

> ## What exactly are you trying to do?
>
> Describe in as much detail as possible.
>
> ## What have you tried so far?
> 
> Describe what you have tried so far.
>
> ## Where else have you checked for solutions?
>
> * [ ] I have read [chezmoi's user guide](https://chezmoi.io/user-guide/command-overview/), and not found the answer.
> * [ ] I have searched [chezmoi's reference guide](https://chezmoi.io/reference/), and not found the answer.
> * [ ] Other, please give details.
>
> ## Output of any commands you've tried with `--verbose` flag
>
> ```console
> $ chezmoi --verbose $COMMAND
> ```
>
> ## Output of `chezmoi doctor`
>
> <details>
> 
> ```console
> $ chezmoi doctor
> ```
>
> </details>
>
> ## Additional context
>
> Add any other context about the problem here.

Of those, the most important are going to be the output of `chezmoi doctor` and the output of the verbose commands (`chezmoi --verbose $COMMAND`).

**Reply by Archie-2021** [reply]
the verbose output for `chezmoi apply -nv` is

```
chezmoi: no encryption
```
And chezmoi doctor output is this:
![chezmoi-doctor.png](https://github.com/twpayne/chezmoi/assets/85870778/dc53590a-d9df-406a-b53f-fc85f9beb159)

**Comment by twpayne** [comment]
You need to provide sufficient information for someone else to reproduce the problem if you want help.

**Comment by Archie-2021** [comment]
> You need to provide sufficient information for someone else to reproduce the problem if you want help.

Finally figured it, forgot to remove .tmpl from my .chezmoi config after copying it. Sorry to bother you.

**Reply by halostatue** [reply]
Glad to hear it's resolved. Normally you wouldn't remove the `.tmpl` but would run `chezmoi init` for it to be templated into place.

**Reply by Archie-2021** [reply]
> Glad to hear it's resolved. Normally you wouldn't remove the `.tmpl` but would run `chezmoi init` for it to be templated into place.

Yeah i had this issue before. I tried chezmoi init but didn't work. No idea why.
